static/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
-  <title>World driven - pull request auto merge</title>
+  <title>World Driven - Pull Request Auto-Merge</title>
```

Comment:
Let's start the naming discussion here, especially because I couldn't make up my mind yet :-)

- World Driven
- worlddriven
- world-driven

Any opinion, if yes, why?

Reply:
I don't like worlddriven - compounds with double letters don't work for me - but could be swayed towards World-Driven I think. I'm envisaging a logo where the dash is the pull request logo and think it works quite well. It also fits in nicely with my preferred auto-merge.

Reply:
K, let's do `World Driven` for now, also thinking that the current domain `worlddriven.org` fits better (otherwise it should be `world-driven.org` :-))

Reply:
I think people would forgive the lack of a hyphen in the URL. I for one wouldn't expect a project called World-Driven to be at world-driven.org. Regardless, World Driven works for me and may in fact be my first choice.